src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/custom.py

```diff
@@ -32,41 +47,68 @@ def cli_consumption_list_usage(cmd, client, billing_period_name=None, top=None,
         filter_to = "properties/usageEnd le \'{}\'".format(end_date.strftime("%Y-%m-%dT%H:%M:%SZ"))
         filter_expression = "{} and {}".format(filter_from, filter_to)
     if top:
-        pages = client.list(scope, expand=expand, filter=filter_expression, top=top)
+        pages = client.list_by_billing_period(billing_period_name, expand=expand, filter=filter_expression, top=top)
         return list(pages.advance_page())
-    return list(client.list(scope, expand=expand, filter=filter_expression))
+    return list(client.list_by_billing_period(billing_period_name, expand=expand, filter=filter_expression))
```

Comment:
`cli_consumption_list_usage()` and `cli_consumption_list_usage_by_billing_period()` essentially do the same thing. It would make much more sense to have one command that just calls the right method from the sdk depending on the presence of "billing_period_name"